src/app/routes/utils/fetchDataFromBFF/index.ts

```diff
@@ -52,14 +54,10 @@ export default async ({
     page,
   });
 
-  const agent = isLocal ? undefined : await getAgent();
-  const timeout = isLocal && BFF_IS_LOCAL ? 60000 : null;
+  const agent = certsRequired(pathname) ? await getAgent() : undefined;
```

Comment:
Think it's answered a few lines below but will this affect our (developers) ability to have local Simorgh and local BFF communicating together in any way?

Reply:
Don't think it should. `certsRequired` checks for `isLocal` as well so it should behave similar to as before.